static/templates/settings/notification_settings_checkboxes.hbs

```diff
@@ -2,8 +2,9 @@
     <label class="checkbox">
         <input type="checkbox" name="{{setting_name}}" id="{{prefix}}{{setting_name}}"
           {{#if is_disabled}}disabled="disabled"{{/if}}
+          {{#is_muted}}disabled="disabled"{{/is_muted}}
```

Comment:
Though this works correctly, we use if statements here `{{#if is_muted}}` instead of directly using these block helpers.

Also we use `disabled` prop now instead of `disabled` attribute as of 498fe285fa. So, I think it should just be `disabled` instead of `disabled="disabled"` in the if condition. @timabbott we inconsistently use both as of now, I think we should fix this to make it consistent.